queries.sql

```diff
@@ -14,3 +14,15 @@ SELECT animals.name, animals.neutered FROM animals WHERE neutered = true;
 SELECT animals.name FROM animals WHERE name != 'Gabumon';
 SELECT animals.name, animals.weight_kg FROM animals 
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- 
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT ROUND(AVG(weight_kg),4) "Average weight" FROM animals;
+SELECT * FROM animals WHERE escape_attempts > 0 AND neutered = true;
+SELECT min(weight_kg), max(weight_kg) FROM animals;
+SELECT AVG(escape_attempts) "Escape attempts"
```

Comment:
* Kindly use the right query here for `What is the average number of escape attempts per animal type of those born between 1990 and 2000?` your query can be narrowed down by using the `Group by species` keyword this would narrow the search to just `animals types` born between `1990` and `2000`